lte/gateway/python/integ_tests/s1aptests/test_attach_detach_rar_activation_reject.py

```diff
@@ -96,21 +98,37 @@ def test_attach_detach_rar_activation_reject(self):
 
         response = self._s1ap_wrapper.s1_util.get_response()
         self.assertEqual(
-            response.msg_type, s1ap_types.tfwCmd.UE_ACT_DED_BER_REQ.value,
+            response.msg_type,
+            s1ap_types.tfwCmd.UE_ACT_DED_BER_REQ.value,
         )
         act_ded_ber_ctxt_req = response.cast(
             s1ap_types.UeActDedBearCtxtReq_t,
         )
-        ded_bearer_rej = s1ap_types.UeActDedBearCtxtRej_t()
-        ded_bearer_rej.ue_Id = req.ue_id
-        ded_bearer_rej.bearerId = act_ded_ber_ctxt_req.bearerId
+        print(
+            "********************** Received Activate Dedicated Bearer Request",
+        )
         time.sleep(15)
+
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type,
+            s1ap_types.tfwCmd.UE_ACT_DED_BER_REQ.value,
+        )
+        print(
```

Comment:
Please add a comment in the test explaining that the timer 3485 expires in 8 seconds, leading to retransmission of dedicated bearer request